### ISSUE TITLE
Sort plugin fires "add" event incorrectly when .push() is called with multiple items

### ISSUE THREAD

**Issue by akagomez** [bitovi/canjs#1509]
For each item that's added to a sorted `can.List` with `.push()` or `.unshift()`, an `add` event is fired with ALL of the items that were passed.

An example: 

```
list.bind('add', function (ev, items, index) {
  console.log('# of items:', items.length); 
}); 
list.push(['a', 'b', 'c']); // -> # of items: 3
                            // -> # of items: 3
                            // -> # of items: 3
```

**Comment by akagomez** [comment]
I have a fix ready. I just need to create a test to go along with it.

**Comment by daffl** [comment]
Closed via #1517
